packages/alfa-rules/src/common/predicate/is-visible.ts

```diff
@@ -77,7 +80,7 @@ function isInvisible(device: Device, context?: Context): Predicate<Node> {
                 // If the element is replaced, visible when empty, or has set dimensions,
                 // it is assumed to be visible
                 nor(isReplaced, isVisibleWhenEmpty, hasDimensions(device)),
-                // otherwise, the element is invisible iff all its children are.
+                // otherwise, the element is invisible if all its children are.
```

Comment:
```suggestion
                // otherwise, the element is invisible iff all its children are.
```
"iff" is a somewhat standard abbreviation for "if and only if"…